airflow/cli/commands/remote_commands/config_command.py

```diff
@@ -64,3 +67,390 @@ def get_value(args):
         print(value)
     except AirflowConfigException:
         pass
+
+
+class RenamedTo(NamedTuple):
+    """Represents a configuration parameter that has been renamed."""
+
+    section: str
+    option: str
+
+
+@dataclass
+class ConfigChange:
+    """Class representing the configuration changes in Airflow 3.0."""
+
+    def __init__(
```

Comment:
One of the main reasons I suggested to use dataclass or attr is that we could reduce `__init__`

Reply:
Right. Changed it